extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -153,6 +153,15 @@ export class AmpAdApiHandler {
           {requestedHeight: height, requestedWidth: width},
           targetOrigin,
           this.is3p);
+    }).catch(() => {
```

Comment:
No need for an extra Promise instance, just pass this as the second parameter to `#then`.